qark/qarkMain.py

```diff
@@ -408,6 +408,7 @@ def nonAutomatedParseArgs():
     advanced_mutual.add_argument("-c", "--codepath", dest="codepath", help="Enter the full path to the root folder containing java source. Required only when --source==2")
 
     optional.add_argument("-e", "--exploit", dest="exploit", help="1 to generate a targeted exploit APK, 0 to skip")
+    optional.add_argument("-x", "--decompile", dest="decompile", help="0 to continue with Static Code Analysis, 1 to EXIT after decompilation")
```

Comment:
We should not need to have people set 0 or 1 for a binary flag. Argparse can handle binary flags (true or false) like this.

`optional.add_argument('--decompile', action='store_true')`